src/matter/common/MessageExchange.ts

```diff
@@ -99,7 +99,11 @@ export class MessageExchange<ContextT> {
         if (messageId === this.receivedMessageToAck?.packetHeader.messageId) {
             // Received a message retransmission but the reply is not ready yet, ignoring
             if (requiresAck) {
-                await this.send(MessageType.StandaloneAck, new ByteArray(0));
+                try {
+                    await this.send(MessageType.StandaloneAck, new ByteArray(0));
+                } catch (error) {
```

Comment:
Can you remove this catch here? Should be catch at top level

Reply:
done

Reply:
(and checked, yes should be catched elsewhere)